### ISSUE TITLE
Scheduling data import/export

### ISSUE THREAD

**Issue by nihlathar** [M66B/XPrivacy#2281]
Hey I have this question, is there a (simple) way to schedule importing settings from a file for a specific app on a hourly basis? To be more specific - I'd like xprivacy to change the fake data it supplies to an app every x minutes, from a pre-exported file (different one each time). I'm pretty sure it can be done but I have no coding background, rooting my device is just about how far my "tech skills" go. Please provide possible method in layman's terms :)

**Comment by M66B** [comment]
Import cannot be automated for security reasons.

For your next question, please read here:
https://github.com/M66B/XPrivacy#support
